nodejs/eks/cluster.ts

```diff
@@ -140,6 +184,26 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
       }, { parent: parent })).role;
     }
 
+    // Create the EKS cluster IAM Role Provider
+    let creationRole: pulumi.Output<aws.iam.Role>;
+    if (args.creationRole !== undefined) {
+        creationRole = pulumi.output(args.creationRole);
+    } else {
+        creationRole = createEntityRole(name, parent);
+    }
+    const eksCreationRoleArn = creationRole.apply(role => role.arn);
```

Comment:
```suggestion
    const eksCreationRoleArn = creationRole.arn;
```

This should work

Reply:
That gives me `Property 'arn' does not exist on type 'OutputInstance<Role>'.`